### PR TITLE
CPTs: Add new a11y labels for WP 4.4.

### PR DESCRIPTION
**Pull request by kraftbj**
https://core.trac.wordpress.org/changeset/34891 added new labels used for hidden headings to help screen readers. Let's add them in.

### CODE REVIEW

File: modules/custom-post-types/comics.php [hunk 1]
```diff
@@ -217,18 +217,21 @@ function register_post_types() {
...
+				'add_new'               => esc_html__( 'Add New',                  'jetpack' ),
+				'add_new_item'          => esc_html__( 'Add New Comic',            'jetpack' ),
+				'edit_item'             => esc_html__( 'Edit Comic',               'jetpack' ),
+				'new_item'              => esc_html__( 'New Comic',                'jetpack' ),
+				'view_item'             => esc_html__( 'View Comic',               'jetpack' ),
+				'search_items'          => esc_html__( 'Search Comics',            'jetpack' ),
+				'not_found'             => esc_html__( 'No Comics found',          'jetpack' ),
+				'not_found_in_trash'    => esc_html__( 'No Comics found in Trash', 'jetpack' ),
+				'filter_items_list'     => esc_html__( 'Filter comics list',       'jetpack' ),
+				'items_list_navigation' => esc_html__( 'Comics list naviagation',  'jetpack' ),
```
Comment (zinigor) [inline modules/custom-post-types/comics.php]: There seems to be a copy-pasted typo in this PR: `naviagation`.

**Comment by zinigor** [comment]
Aside from one multiplied typo, this looks good!

**Comment by kraftbj** [comment]
Fixed typo. Marking as Review Done per @zinigor's look. :)

File: modules/custom-post-types/comics.php [hunk 2]
```diff
@@ -217,18 +217,21 @@ function register_post_types() {
...
+				'add_new'               => esc_html__( 'Add New',                  'jetpack' ),
+				'add_new_item'          => esc_html__( 'Add New Comic',            'jetpack' ),
+				'edit_item'             => esc_html__( 'Edit Comic',               'jetpack' ),
+				'new_item'              => esc_html__( 'New Comic',                'jetpack' ),
+				'view_item'             => esc_html__( 'View Comic',               'jetpack' ),
+				'search_items'          => esc_html__( 'Search Comics',            'jetpack' ),
+				'not_found'             => esc_html__( 'No Comics found',          'jetpack' ),
+				'not_found_in_trash'    => esc_html__( 'No Comics found in Trash', 'jetpack' ),
+				'filter_items_list'     => esc_html__( 'Filter comics list',       'jetpack' ),
+				'items_list_navigation' => esc_html__( 'Comics list navigation',  'jetpack' ),
```
Comment (dereksmart) [inline modules/custom-post-types/comics.php]: The typo fix misaligned the `jetpack` columns with a missing space :cheesy:

**Pull request merged**